src/Controls/src/Core/ILineHeightElementInternal.cs

```diff
@@ -0,0 +1,15 @@
+#nullable disable
+using System;
+using System.ComponentModel;
+
+namespace Microsoft.Maui.Controls.Internals
```

Comment:
I have duplicated this one. I wanted to get rid of the "Internals" in the namespace, but since we need to provide a transition path, we want to keep the one that is available right now, and provide the new public one in the new namespace. 

Once we remove `InternalsVisibleTo` we can remove this one as well. I will make sure to note that in the issue to track this.